compiler/luci/pass/src/RemoveRedundantReshape.cpp

```diff
@@ -48,18 +48,25 @@ bool remove_redundant_reshape(luci::CircleNode *node)
 namespace luci
 {
 
-// Bypass redundant reshape nodes:
-//
-//      input                      input  ---+
-//        |                          |       |
-//        V                          V       |
-//     Reshape       becomes      Reshape    |
-//   (pred_node)                (pred_node)  |
-//        |                                  |
-//        V                                  |
-//     Reshape                    Reshape  <-+
-//  (target_node)              (target_node)
-
+/**
+ * BEFORE
+ *
+ *      [CircleNode]
+ *            |
+ *    [CircleReshape_1]
+ *            |
+ *    [CircleReshape_2]
+ *            |
+ *      [CircleNode]
+ *
+ * AFTER
+ *
+ *      [CircleNode]
+ *            |
+ *    [CircleReshape_2]
+ *            |
+ *      [CircleNode]
+ **/
```

Comment:
Previous diagram had `Reshape` after the Pass but new diagram `CircleReshape_1` is gone.
Please revise this diagram to reflect what this Pass does.

Reply:
Oh, I forget to add. I will add it.